install/kubernetes/helm/istio/templates/clusterrole.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.global.istioRemote }}
```

Comment:
is this correct that it only applied to remote?

Reply:
the other change LG to me.

Reply:
Correct, the `clusterrole, clusterrolebinding and serviceaccount` are used for `pilot` to `watch/list` resources in remote cluster, only applied in multicluster topology through the `VPN connectivity`